src/FakeItEasy.Analyzer/Properties/AssemblyInfo.cs

```diff
@@ -1,11 +1,10 @@
 ﻿using System;
-using System.Reflection;
 using System.Runtime.CompilerServices;
 
-[assembly: AssemblyProduct("FakeItEasy.Analyzer.VisualBasic")]
-[assembly: AssemblyTitle("FakeItEasy.Analyzer.VisualBasic")]
-[assembly: AssemblyDescription("Diagnostic analyzers to warn about incorrect usage of FakeItEasy in VB.NET code.")]

```

Comment:
Ah, I see... I had `<GenerateAssemblyInfo>false</GenerateAssemblyInfo>` in the csproj (because of `CommonAssemblyInfo.cs`), so the attributes weren't generated.